dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m

```diff
@@ -43,7 +43,8 @@ - (void)testFullScreenWarm {
     XCTAssertTrue([app.navigationBars[@"Flutter iOS Demos Home"] waitForExistenceWithTimeout:60.0]);
 }
 
-- (void)testFlutterViewWarm {
+// TODO(dnfield): this test has become flaky, fix https://github.com/flutter/flutter/issues/70630 and re-enable.
+- (void)skip_testFlutterViewWarm {
```

Comment:
Can you instead uncheck it from the scheme instead of the `skip_` prefix, but leave the comment?  (I'm happy to do it if needed)
<img width="896" alt="Screen Shot 2020-11-19 at 2 05 51 PM" src="https://user-images.githubusercontent.com/682784/99729863-64285280-2a70-11eb-9962-c9e69500dae1.png">

Reply:
I'm in the middle of reinstalling Xcode - you might be able to get to this more quickly  than I can right now if you don't mind.

Reply:
ok I'm on it now, install finished